setup.py

```diff
@@ -58,6 +58,7 @@ def run_tests(self):
     dogpile.cache = dogpile.cache.plugins.mako_cache:MakoPlugin
     """,
     zip_safe=False,
+    install_requires=['decorator<5'],
```

Comment:
would rather remove the <5 here, I wouldn't worry that some major breaking change is coming in decorator 5